src/data/data1d/Data1DManager.js

```diff
@@ -44,23 +43,23 @@ export class Data1DManager {
         original: data,
       },
     });
-    // console.log(datum1D);
+
     return datum1D;
   };
 }
 
 function getData(spectra) {
   let x =
-    spectra[0] && spectra[0].data && spectra[0].data && spectra[0].data.x
-      ? spectra[0].data.x
+    spectra[0] && spectra[0].data && spectra[0].data[0].x
+      ? spectra[0].data[0].x
```

Comment:
This change would break the current version of jcamp converter. In which case to we have more than one element in the data array ? If it never happens the Bruker converter should be changed

Reply:
So, I will upgrade jcampconverter dependency in brukerconverter